dashboard/app/controllers/foorm_library_editor_controller.rb

```diff
@@ -0,0 +1,22 @@
+# Foorm Library Editor is only available on levelbuilder or test, for those with levelbuilder permissions.
+class FoormLibraryEditorController < ApplicationController
+  before_action :require_levelbuilder_mode_or_test_env
+  before_action :authenticate_user!
+  authorize_resource class: false
+
+  # GET '/foorm/library_editor/'
```

Comment:
Note: this doesn't match what is in the routes. For consistency should this be `foorm/library/editor`? Or the form one could be `foorm/forms_editor`